src/sentry/objectstore/__init__.py

```diff
@@ -1,3 +1,5 @@
+from sentry import options as options_store
 from sentry.objectstore.service import ClientBuilder
 
-attachments = ClientBuilder("attachments")
+options = options_store.get("objectstore.config")
+attachments = ClientBuilder(options["base_url"], "attachments")
```

Comment:
maybe pass in all the `options`. previously, we also had a shared secret there, so it might be good to pass that as well.